### PR TITLE
Organize TrackImport Directory and Fix Tests

### PR DESCRIPTION
**Pull request by Qstick**
#### Database Migration
NO

#### Description
Organize `TrackImport` Directory and Fix Tests for Import Specifications

#### Todos
- [x] Tests

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/NzbDrone.Core/MediaFiles/TrackImport/IImportDecisionEngineSpecification.cs [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+using NzbDrone.Core.DecisionEngine;
 using NzbDrone.Core.Parser.Model;
 
-namespace NzbDrone.Core.MediaFiles.EpisodeImport
+namespace NzbDrone.Core.MediaFiles.TrackImport
 {
     public interface IImportDecisionEngineSpecification
     {
-        Decision IsSatisfiedBy(LocalEpisode localEpisode);
+        //Decision IsSatisfiedBy(LocalEpisode localEpisode);
```
Comment (danielunderwood) [inline src/NzbDrone.Core/MediaFiles/TrackImport/IImportDecisionEngineSpecification.cs]: Remove?

File: src/NzbDrone.Core/MediaFiles/TrackImport/ImportApprovedEpisodes.cs [hunk 2]
```diff
@@ -12,10 +12,10 @@
 using NzbDrone.Core.Qualities;
 using NzbDrone.Core.Download;
 using NzbDrone.Core.Extras;
-using NzbDrone.Core.MediaFiles.TrackImport;
 
-namespace NzbDrone.Core.MediaFiles.EpisodeImport
+namespace NzbDrone.Core.MediaFiles.TrackImport
 {
+    [Obsolete("Used by Sonarr, not by Lidarr")]
```
Comment (danielunderwood) [inline src/NzbDrone.Core/MediaFiles/TrackImport/ImportApprovedEpisodes.cs]: I guess this means things break if we try to take it out now?

File: src/NzbDrone.Core/MediaFiles/TrackImport/IImportDecisionEngineSpecification.cs (hunk 1 above)
Comment (Qstick) [reply src/NzbDrone.Core/MediaFiles/TrackImport/IImportDecisionEngineSpecification.cs]: Will remove

File: src/NzbDrone.Core/MediaFiles/TrackImport/ImportApprovedEpisodes.cs (hunk 2 above)
Comment (Qstick) [reply src/NzbDrone.Core/MediaFiles/TrackImport/ImportApprovedEpisodes.cs]: Yep, will be able to pull out when completed handling goes in after this pr

**Pull request merged**